pootle/apps/pootle_app/management/commands/__init__.py

```diff
@@ -17,6 +17,22 @@
 from pootle_translationproject.models import TranslationProject
 
 
+class SkipChecksMixin(object):
+    def check(self, app_configs=None, tags=None, display_num_errors=False,
+              include_deployment_checks=False):
+        skip_tags = getattr(self, 'skip_system_check_tags', None)
+        if skip_tags is not None:
+            from django.core.checks.registry import registry
+            tags = registry.tags_available()
```

Comment:
Just wanted to suggest the same thing. And having to say that readability is related thing sometimes.